stories/Components/UIcomponents/Cards/ContentCard/ContentCardWithOutImage.jsx

```diff
@@ -7,40 +7,28 @@ import { P } from '../../../../Atom/Base-typography/Paragraph/Paragraph';
 
 const cls = (...classes) => ((classes.filter(Boolean).length > 0) ? classes.filter(Boolean).join(' ') : null);
 
-export const emphasize_options = {
-  true: 'card-emphasize',
-  false: 'card',
+export const hovercolors_options = {
+  default: '',
+  yellow: 'yellow',
+  red: 'red',
+  blue: 'blue',
+  green: 'green',
 };
 
 export const ContentCardWithOutImage = ({
-  data, Hovercolors, Emphasize,
+  data, Hovercolors,
 }) => {
-  let emphasize_variant = emphasize_options[`${Emphasize}`];
+  let hovercolors_variant = hovercolors_options[`${Hovercolors}`];
   return (
     <div className="grid-x grid-margin-x">
       {data.map((item, index) => (
         <div key={index} className="cell medium-4">
-          <div className={cls('content-card', `${emphasize_variant}`, `${Hovercolors}`)}>
+          <div className={cls('content-card', 'card', `${hovercolors_variant}`)}>
```

Comment:
@Shubham-Sharma-77 Can we remove this class `card` I don't see it getting used anywhere.